### PR TITLE
Created a preliminary coordinates file with WCS Function

### PR DESCRIPTION
**Pull request by stefannelson**
Created a preliminary coordinates file and added my Pixel to WCS conversion function. Still a (very) rough draft as of right now. A concern I already have is that the way this is currently set up; it needs 2 arrays, one for each axis. 

@stottsco What are you expecting to pass into the function when you call this in your photometry function?

### CODE REVIEW

File: glowing_waffles/coordinates.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function, division
+import numpy as np
+
+from ccdproc import CCDData
+from astropy.wcs import WCS
+
+def convert_pixel_wcs(ccd_image, lon_or_ra, lat_or_dec, isPix=True):
```
Comment (mwcraig) [inline glowing_waffles/coordinates.py]: Please change `isPix` → `is_pix`

File: glowing_waffles/coordinates.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def convert_pixel_wcs(ccd_image, lon_or_ra, lat_or_dec, isPix=True):
         Represents the lat/dec axis, depending on what is
         passed in.
 
-    isPix: Bool
+    is_pix: Bool
```
Comment (mwcraig) [inline glowing_waffles/coordinates.py]: `Bool` → `bool`

File: glowing_waffles/coordinates.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ def convert_pixel_wcs(ccd_image, lon_or_ra, lat_or_dec, isPix=True):
     and the second lat/Dec.
 
     """
-    if isPix:
+    if is_pix:
         return ccd_image.wcs.all_pix2world(lon_or_ra, lat_or_dec, 0)
```
Comment (mwcraig) [inline glowing_waffles/coordinates.py]: Maybe you should add an argument to the function (with default value of 0) that can be passed in for the origin argument (which you now have set to 0).

**Comment by mwcraig** [comment]
A couple more minor changes, then ready to merge.

File: glowing_waffles/coordinates.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function, division
+import numpy as np
+
+from ccdproc import CCDData
+from astropy.wcs import WCS
+
+def convert_pixel_wcs(ccd_image, lon_or_ra, lat_or_dec, is_pix=True):
```
Comment (mwcraig) [inline glowing_waffles/coordinates.py]: Could you please also add an `origin` argument, that defaults to zero, and then pass that argument in to the conversion routines, as indicated below?

File: glowing_waffles/coordinates.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        Assumes pixel coordinates by default.
+
+    Returns
+    -------
+    Returns two numpy.ndarray objects, the first being lon/RA,
+    and the second lat/Dec.
+
+    """
+    if is_pix:
+        return ccd_image.wcs.all_pix2world(lon_or_ra, lat_or_dec, 0)
```
Comment (mwcraig) [inline glowing_waffles/coordinates.py]: pass origin argument in here instead of 0.

File: glowing_waffles/coordinates.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    else:
+        return ccd_image.wcs.all_world2pix(lon_or_ra, lat_or_dec, 0)
```
Comment (mwcraig) [inline glowing_waffles/coordinates.py]: Same thing for the 0 here.

File: glowing_waffles/coordinates.py [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        An array of coordinates (or a single coordinate).
+        Represents the lon/ra axis, depending on what is
+        passed in.
+
+    lat_or_dec: numpy.ndarray (or float)
+        An array of coordinates (or a single coordinate).
+        Represents the lat/dec axis, depending on what is
+        passed in.
+
+    is_pix: Bool
```
Comment (mwcraig) [inline glowing_waffles/coordinates.py]: `Bool` should be lower case.

File: glowing_waffles/coordinates.py [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    """
+    Takes either pixel or World Coordinate System (RA/Dec)
+    coordinates and converts to the other type.
+
+    Parameters
+    ----------
+    ccd_image: CCDData object
+        Image which has the coordinates to convert.
+
+    lon_or_ra: numpy.ndarray (or float)
```
Comment (stottsco) [inline glowing_waffles/coordinates.py]: add backticks around `numpy.ndarray'

File: glowing_waffles/coordinates.py [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+        An array of coordinates (or a single coordinate).
+        Represents the lon/ra axis, depending on what is
+        passed in.
+
+    lat_or_dec: numpy.ndarray (or float)
```
Comment (stottsco) [inline glowing_waffles/coordinates.py]: and here

File: glowing_waffles/coordinates.py [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+        Represents the lat/dec axis, depending on what is
+        passed in.
+
+    isPix: Bool
+        Used to determine what coordinates are being inputted.
+        Assumes pixel coordinates by default.
+
+    Returns
+    -------
+    Returns two numpy.ndarray objects, the first being lon/RA,
```
Comment (stottsco) [inline glowing_waffles/coordinates.py]: and again here

File: glowing_waffles/coordinates.py [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+    and the second lat/Dec.
+
+    """
+    if isPix:
```
Comment (stottsco) [inline glowing_waffles/coordinates.py]: is_pix

**Pull request merged**